<launch>
  
  <arg name="calib_path" default="/home/z/Desktop/bag/a.yaml" />
  <arg name="input_camera_name_space" default="cam0"/>

  <rosparam file="$(arg calib_path)"/>

  <node name="image_undistort_node" pkg="image_undistort" type="image_undistort_node" output="screen" clear_params="true">
    <param name="input_camera_name_space" value="$(arg input_camera_name_space)"/>

    <remap from="/cam0/image_raw" to="/uvc_camera/cam_0/image_raw"/>

    <!--optional arguments, currently all set to their default values, see image_undistort.h for further infomation -->
    <param name = "input_camera_info_from_yaml" value = "true"/>
    <param name = "queue_size" value = "100"/>
    <param name = "process_image" value = "true"/>
    <param name = "undistort_image" value = "true"/>
    <param name = "process_every_nth_frame" value = "1"/>
    <param name = "zoom" value = "1.0"/>
  </node>

</launch>